projects/plugins/jetpack/modules/contact-form/js/form-styles.js

```diff
@@ -0,0 +1,78 @@
+jQuery( function ( $ ) {
+	const EDITOR_SELECTOR = '[data-type="jetpack/contact-form"]';
+	const FRONTEND_SELECTOR = '.wp-block-jetpack-contact-form-container';
```

Comment:
What problem are we trying to solve with the suggested change?
I'm sorry, I didn't get it 😕

Reply:
Ah, that was me getting confused, sorry about that! I saw `block` in the name and wasn't sure it applied to the old contact form shortcode as well. Turns out it does, since the block and the shortcode share the same implementation there. You can ignore me :)